### DISCUSSION TITLE
How to save photo in QField app and be able to show it in QGIS (by using Android local file)

### DISCUSSION THREAD

**Discussion by zacharylee1204** [opengisch/QField#4120] [Q&A]
Dear all,

I have a difficult time to export the field data (photo is taken in the app). I can see the photo on the app. However, I am not able to see it when I transfer it back to my computer (by USB cable). May I ask how to do that?

Thank you.

Zachary

**Comment by zacharylee1204** [comment]
Anybody could give me a hand? thank you very much

**Comment by VxTedxV** [comment]
Your photos should have been saved in a 'DCIM' folder next to your project file. Access only from computer (not from Android).

**Comment by r3tlove** [comment]
here is how i am doing it
/map_project_directory/mymap.qgs
/map_project_directory/qfield_record-pts/mysurveylayer.gpkg
/map_project_directory/qfield_record-pts/photos  #wher my photo (attachment) go to

so then i want my photos to end up in a subdirectory of qfield_record-pts
so i put this in the qfield attachment name

'qfield_record-pts/photos/tlove_'||to_string(format_date(now(),'yyyy-MM-dd_hh_mm_ss_zzz'))||'.jpg'

copy the whole subdirectory of qfield_record-pts back to your computer and they should end up easily accessible since its a simple relative path. i use the attachment widget to get them to show up with the Info tool..

![image](https://github.com/opengisch/QField/assets/21207835/a30571a1-e46a-4693-8ffc-75ebf0ba7f48)
